molgenis-js/src/main/resources/js/script-evaluator.js

```diff
@@ -1,452 +1,444 @@
 /**
  * Evaluates a script.
  *
- * @param script
- *            the script
- * @param entity
- *            the entity
+ * @param script the script
+ * @param entity object mapping attribute name key to attribute value
  * @returns the evaluated script result
  */
 function evalScript (script, entity) {
-  function Attribute (value) {
+    // make functions available to the script in local scope
+    var $ = MagmaScript.$.bind(entity)
```

Comment:
IntelliJ reports various warnings in the function related. Could you fix or suppress them?